Libs/Particles/ParticleSystem.cpp

```diff
@@ -5,12 +5,10 @@ ParticleSystem::ParticleSystem()
 
 }
 
-bool ParticleSystem::LoadParticles(const std::vector<std::string> &_paths)
+void ParticleSystem::LoadParticles(const std::vector<std::string> &_paths)
 {
-
   if (_paths.empty()) {
-    std::cerr << "No filenames passed to readparticlesystem; returning false." << std::endl;
-    return false;
+    throw std::runtime_error("No filenames passed to readparticlesystem");
```

Comment:
This reinforces my previous comment since loading particles is now a non-optional operation.

Reply:
Loading is non-optional. This check is just to ensure that we haven't been passed an _empty_ list of files